pkg/eib/eib.go

```diff
@@ -98,14 +98,19 @@ func appendElementalRPMs(ctx *image.Context) {
 
 	log.AuditInfo("Elemental registration is configured. The necessary RPM packages will be downloaded.")
 
-	appendRPMs(ctx, []image.AddRepo{
-		{
-			URL: ctx.ArtifactSources.Elemental.RegisterRepository,
-		},
-		{
-			URL: ctx.ArtifactSources.Elemental.SystemAgentRepository,
-		},
-	}, combustion.ElementalPackages...)
+	if ctx.ImageDefinition.APIVersion == "1.0" {
```

Comment:
This isn't a good indicator. You can build SL Micro 6.0 images with both API versions 1.0 and 1.1. Do we bite the bullet and only allow Elemental builds for 6.0 in EIB v1.1?